yt_dlp/extractor/twitch.py

```diff
@@ -28,7 +28,7 @@
     unified_timestamp,
     update_url_query,
     url_or_none,
-    urljoin,
+    urljoin, str_or_none,
```

Comment:
don't be a monster....put it on a newline 😄 
also perhaps sort it alphabetically

Reply:
sorry, it happened on my other pr too :D
Be it my excuse that I didn't do this intentionally, PyCharm's quick import does this